pysmt/test/test_rewritings.py

```diff
@@ -83,6 +83,60 @@ def test_prenex_examples(self):
                         ok = not logic.quantifier_free
                     self.assertTrue(ok)
 
```

Comment:
Can you add a test in which you explicit the result of the ackermanization? Right now these tests would pass if the ackermanizatoin function were to return always ``FALSE()``, since both examples seem to be unsat.

Reply:
I have added a test (not pushed yet) that uses all of `example_formulae`, whenever applicable. This test covers both sat and unsat. Do you think this suffices?